e2e/e2e.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path"
+	"time"
+
+	"github.com/getoutreach/gobox/pkg/sshhelper"
+	"github.com/go-git/go-billy/v5/memfs"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+var virtualDeps = map[string][]string{
+	// TODO: Put a service.yaml in flagship with this
+	"flagship": {
```

Comment:
so is this temporary? because this repository is open source and all of these are private projects

Reply:
yeah, this is temporary, I can't put it in the flagship today because it'd enable a lot of things that'd cause bad UX here. I plan on removing it soon, I also doubt this will be used anytime soon